libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/Navigation/Navigation.tsx

```diff
@@ -102,18 +108,42 @@ export class Navigation extends React.Component<INavigationProps> {
       }
     }
     const classNames = navigationStyles();
+    const messageBarStyles: IMessageBarStyles = {
+      content: classNames.content,
+      dismissal: classNames.dismissal,
+      dismissSingleLine: classNames.dismissSingleLine,
+      iconContainer: classNames.iconContainer,
+      text: classNames.text
+    };
     return (
       <div className={classNames.navigation}>
-        <div className={classNames.breadcrumb}>
-          <Breadcrumb
-            items={items}
-            maxDisplayedItems={10}
-            ariaLabel="Navigation"
-            overflowAriaLabel="More links"
-            onRenderItem={this._onRenderItem}
-            styles={breadcrumbStyle}
-          />
-        </div>
+        <Stack horizontal horizontalAlign="space-between">
+          <Stack.Item>
+            <div className={classNames.breadcrumb}>
+              <Breadcrumb
+                items={items}
+                maxDisplayedItems={10}
+                ariaLabel="Navigation"
+                overflowAriaLabel="More links"
+                onRenderItem={this._onRenderItem}
+                styles={breadcrumbStyle}
+              />
+            </div>
+          </Stack.Item>
+          {this.props.showMessageBar && (
+            <Stack.Item>
+              <MessageBar
+                messageBarType={MessageBarType.success}
+                isMultiline={false}
+                onDismiss={() => this.props.closeMessageBar()}
+                className={classNames.messageBar}
```

Comment:
why do we need both className and styles? can we merge them to one?

Reply:
+1 on this

Reply:
done!  Moved the className style values to be the root of the styles property.  Now className is removed.